src/uct/ib/base/ib_device.c

```diff
@@ -1294,17 +1294,19 @@ const char* uct_ib_ah_attr_str(char *buf, size_t max,
     char *p    = buf;
     char *endp = buf + max;
 
-    snprintf(p, endp - p, "dlid=%d sl=%d port=%d src_path_bits=%d",
+    snprintf(p, endp - p, "dlid=0x%x sl=0x%x port=%d src_path_bits=%d",
```

Comment:
why need to print LID and SL as hex? in other places they are printed as decimal

Reply:
I'll keep the original print format. The original idea is to keep use similar format as GID(hex) to print LID.